api_docs/markdowns/api/bql/bql-functions.md

```diff
@@ -0,0 +1,96 @@
+# BQLFunction
+
+## Usage
+
+Using BQL, function calls can have different formats.
+For a function with only one argument, it can be called as:
+```JSON
+{"function_name": "field_name"}
+```
+For functions with more than one argument, the keywords `"function"` and `"args"` must be used:
+```JSON
+{
+    "function": "function_name",
+    "args": [
+        "field_name_1",
+        "field_name_2"
+    ]
+}
+```
+
+## Functions
+
+### Mathematical Functions
+
+The Mathematical Functions output a numerical value or `null`. 
+
+- `add`
```

Comment:
Don't we want to have a few words of description for each function?

Reply:
I don't know. Somehow yes, somehow no.

It would be nice to have a common presentation for all functions so that the user expects a certain format when looking for a function (of input, output and description for example).
But at the same time, I'm not fond of having obvious information like:
```
Function: "add"
Description: "Adds values together"
```

What do you think?